CHANGELOG.md

```diff
@@ -4,6 +4,123 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.3.0] - 2022-11-09
+
+## 🚀 Features
+
+### Add support for DHAT-based heap profiling ([PR #1829](https://github.com/apollographql/router/pull/1829))
+
+The [dhat-rs](https://github.com/nnethercote/dhat-rs) crate provides [DHAT](https://www.valgrind.org/docs/manual/dh-manual.html)-style heap profiling. We have added two compile-time features, `dhat-heap` and `dhat-ad-hoc`, which leverage this ability.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/1829
+
+### Add `trace_id` in logs to correlate entries from the same request ([Issue #1981](https://github.com/apollographql/router/issues/1981))
+
+A `trace_id` is now added to each log line to help correlate log entries to specific requests.  The value for this property will be automatically inherited from any enabled distributed tracing headers, such as those listed in our [Tracing propagation header](https://www.apollographql.com/docs/router/configuration/tracing/#propagation) documentation (e.g., Jaeger, Zipkin, Datadog, etc.).
+
+In the event that a `trace_id` was not inherited from a propagated header, the Router will originate a `trace_id` and also propagate that value to subgraphs to enable tracing in subgraphs.
```

Comment:
It propagates `trace_id` using tracing propagation header only if it is enabled in the configuration mentioned here https://www.apollographql.com/docs/router/configuration/tracing/#propagation

Reply:
@bnjjj I tried to make that clear in my phrasing of this:

> be automatically inherited from any enabled distributed tracing headers, such as those listed in our [Tracing propagation header](https://www.apollographql.com/docs/router/configuration/tracing/#propagation) documentation

...but do you have suggestions on how to make it more clear?

Reply:
Maybe this?

```suggestion
In the event that a `trace_id` was not inherited from a propagated header — or header propagation is not enabled — the Router will originate a `trace_id` and also propagate that value to subgraphs to enable tracing in subgraphs.
```

Reply:
```suggestion
In the event that a `trace_id` was not inherited from a propagated header, the Router will originate a `trace_id` and also propagate that value to subgraphs to enable tracing in subgraphs (only if configured with [Tracing propagation header](https://www.apollographql.com/docs/router/configuration/tracing/#propagation)).
```

Reply:
I think I see what you're talking about.  How about entirely removing the last bit of that sentence:

```suggestion
In the event that a `trace_id` was not inherited from a propagated header, the Router will originate a `trace_id`.
```